applications/CableNetApplication/tests/ring_set_up_dem_fem/MaterialsDEM.json

```diff
@@ -0,0 +1,51 @@
+{
+    "materials":[{
+        "material_name": "mat1",
+        "material_id": 1,
+        "properties":{
+            "PARTICLE_DENSITY": 3000.0,
+            "YOUNG_MODULUS": 1.0e5,
+            "POISSON_RATIO": 0.20
+        }
+    },{
+        "material_name": "mat2",
+        "material_id": 2,
+        "properties":{
+            "YOUNG_MODULUS": 1e20,
+            "POISSON_RATIO": 0.25,
+            "COMPUTE_WEAR": false
+        }
+    }],
+    "material_relations":[{
+        "material_names_list":["mat1", "mat1"],
+        "material_ids_list":[1, 1],
+        "properties":{
+            "COEFFICIENT_OF_RESTITUTION": 0.2,
+            "STATIC_FRICTION": 0.9999999999999999,
+            "DYNAMIC_FRICTION": 0.9999999999999999,
+            "FRICTION_DECAY": 500,
+            "ROLLING_FRICTION": 0.01,
+            "ROLLING_FRICTION_WITH_WALLS": 0.01,
+            "DEM_DISCONTINUUM_CONSTITUTIVE_LAW_NAME": "DEM_D_Hertz_viscous_Coulomb"
+        }
+    },{
+        "material_names_list":["mat1", "mat2"],
+        "material_ids_list":[1, 2],
+        "properties":{
+            "COEFFICIENT_OF_RESTITUTION": 0.2,
+            "STATIC_FRICTION": 0.7886751345948129,
+            "DYNAMIC_FRICTION": 0.7886751345948129,
+            "FRICTION_DECAY": 500,
+            "ROLLING_FRICTION": 0.01,
+            "ROLLING_FRICTION_WITH_WALLS": 0.01,
+            "SEVERITY_OF_WEAR": 0.001,
+            "IMPACT_WEAR_SEVERITY": 0.001,
+            "BRINELL_HARDNESS": 200.0,
+            "DEM_DISCONTINUUM_CONSTITUTIVE_LAW_NAME": "DEM_D_Hertz_viscous_Coulomb"
+        }
+    }],
+    "material_assignation_table":[
```

Comment:
```suggestion
    "material_assignment_table":[
```

Reply:
After reading several dictionaries it seems both options can be used. I will leave assignation for less noise.